src/tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "lib": ["es2020", "dom", "dom.iterable"],
     "jsx": "react-jsx",
     "jsxImportSource": "preact",
-    "module": "commonjs",
+    "module": "es2020",
+    "moduleResolution": "node",
```

Comment:
The `module` configuration update is needed to use `import.meta`. The `moduleResolution` change is needed to allow the use of Node-style import paths (eg. without a file extension, and lookup in `node_modules` if the path doesn't start with `./`) together with `es2020`.